Source/Blazorise/JsOptions/ColorPickerJsOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Blazorise;
+
+public record ColorPickerJsOptions(
+    string Default,
+    string[] Palette,
+    bool ShowPalette,
+    bool HideAfterPaletteSelect,
+    bool ShowClearButton,
+    bool ShowCancelButton,
+    bool ShowOpacitySlider,
+    bool ShowHueSlider,
+    bool ShowInputField,
+    bool Disabled,
+    bool ReadOnly,
+    IReadOnlyDictionary<string, string> Localization,
+    string ColorPreviewElementSelector,
+    string ColorValueElementSelector
+    );
+
+public record OptionChange(bool Changed, bool Value);
```

Comment:
Maybe rename this to 

- `JsOptionChange`
- `JsOptionUpdate`
- something else?

Reply:
make sense.